servers/sunrpc/handshake_prog.go

```diff
@@ -6,7 +6,8 @@ import (
 	"path"
 
 	"github.com/gluster/glusterd2/utils"
-
+	"github.com/gluster/glusterd2/servers/sunrpc/program"
```

Comment:
Usually, I'd prefer the plugins/rpc programs to not import anything from  `servers/sunrpc`. This will cleanly avoid cyclic imports without having to create subpackages. As a coincidence I [moved](https://github.com/prashanthpai/sunrpc/commit/628e7a95e2aa0d3b8c8723cdd2f3297518b9d341#diff-4518f35e76c810981e52e8c8f90c4fcb) the `Procedure` struct to `sunrpc` package today. Hope this helps. Let me know what you think.